posts/2021-03-13-microprofile-config-2.0.adoc

```diff
@@ -240,7 +254,7 @@ The behaviour for "empty" and "special" config property values has been updated:
 * A value is considered to be "empty" if the link:https://download.eclipse.org/microprofile/microprofile-config-2.0/apidocs/org/eclipse/microprofile/config/spi/Converter.html[Converter] being used considers it to be "empty". For example: 
 ** All Converters consider `""`, the empty String, to be empty.
```

Comment:
Technically the spec does not require there to be a Converter for String (to convert from String to String). So maybe you need to add a bullet that simply says something like "A value is considered to be empty if the raw String is empty and is not converted"? Not sure if I am making this over-complicated?

Reply:
Hmm I think it would be more clear without the additional comment